pennylane_forest/converter.py

```diff
@@ -585,6 +587,10 @@ def load_program(program: pyquil.Program):
     Every variable that is present in the Program and that is not used as the target
     register of a measurement has to be provided in the ``parameter_map`` of the template.
 
+    .. warning::
+
+        Converting from Forest is not supported with PennyLane version 0.14.0.
```

Comment:
Given that we raise an error above, is there any version of PL that this would work with?

Reply:
Updated the message, it should work with PL version `<0.14.0`